src/main/java/com/erigitic/commands/JobCommand.java

```diff
@@ -25,67 +25,264 @@
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
 import java.util.Optional;
 
 public class JobCommand implements CommandExecutor {
-    private AccountManager accountManager;
-    private TEJobManager teJobManager;
+
+    public static CommandSpec commandSpec(TotalEconomy totalEconomy) {
+        return CommandSpec.builder()
+                .child(Set.commandSpec(totalEconomy), "set", "s")
+                .child(Toggle.commandSpec(totalEconomy), "toggle", "t")
+                .child(Info.commandSpec(totalEconomy), "info", "i")
+                .child(Reload.commandSpec(totalEconomy), "reload")
+                .description(Text.of("Display list of jobs."))
+                .permission("totaleconomy.command.job")
+                .arguments(GenericArguments.none())
+                .executor(new JobCommand(totalEconomy))
+                .build();
+    }
+
+    private TotalEconomy totalEconomy;
 
     public JobCommand(TotalEconomy totalEconomy) {
-        accountManager = totalEconomy.getAccountManager();
-        teJobManager = totalEconomy.getTEJobManager();
+        this.totalEconomy = totalEconomy;
     }
 
     @Override
     public CommandResult execute(CommandSource src, CommandContext args) throws CommandException {
         if (src instanceof Player) {
             Player player = ((Player) src).getPlayer().get();
+            String jobName = totalEconomy.getTEJobManager().getPlayerJob(player);
+            player.sendMessage(Text.of(TextColors.GRAY, "Your current job is: ", TextColors.GOLD, jobName));
+            player.sendMessage(Text.of(TextColors.GRAY, totalEconomy.getTEJobManager().titleize(jobName),
+                    " Level: ", TextColors.GOLD, totalEconomy.getTEJobManager().getJobLevel(jobName, player)));
+            player.sendMessage(Text.of(TextColors.GRAY, totalEconomy.getTEJobManager().titleize(jobName),
+                    " Exp: ", TextColors.GOLD, totalEconomy.getTEJobManager().getJobExp(jobName, player), "/", totalEconomy.getTEJobManager().getExpToLevel(player), " exp\n"));
+            player.sendMessage(Text.of(TextColors.GRAY, "Available Jobs: ", TextColors.GOLD, totalEconomy.getTEJobManager().getJobList()));
+            return CommandResult.success();
+        } else {
+            throw new CommandException(Text.of("You can't have a job!"));
+        }
+    }
+
+    //// === === === === === SET === === === === === ////
+
+    public static class Set implements CommandExecutor {
+
+        public static CommandSpec commandSpec(TotalEconomy totalEconomy) {
+            return CommandSpec.builder()
+                    .description(Text.of("Set your job"))
+                    .permission(TEPermissions.JOB_SET)
+                    .executor(new Set(totalEconomy))
+                    .arguments(
+                            GenericArguments.string(Text.of("jobName")),
+                            GenericArguments.requiringPermission(
```

Comment:
Be sure to document the addition of `/job set [job] [player]` in the changelog under v1.6.0.